### PR TITLE
Dockerize app

### PR DESCRIPTION
**Pull request by ankitjilka**
I was add dockerfile and compose file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ python -m pip install venv
 ```bash
 . flask_note_env/bin/activate
 ```
-
+===================================================
```
Comment (zeelrupapara) [inline README.md]: remove this line

File: README.md [hunk 2]
```diff
@@ -31,11 +31,14 @@ python -m pip install -r requirements.txt
 export FLASK_APP=main.py
 export FLASK_ENV=development
 ```
+====================================================
```
Comment (zeelrupapara) [inline README.md]: remove this line

File: README.md [hunk 3]
```diff
@@ -31,11 +31,14 @@ python -m pip install -r requirements.txt
...
 #### Now, Run flask app,
 `Recommend: Before run application apply migration`
 ```bash
 flask run
 ```
+
+====================================================`
```
Comment (zeelrupapara) [inline README.md]: remove this line

File: README.md [hunk 4]
```diff
@@ -54,3 +57,22 @@ flask db migrate -m "init migration"
 ```python
 flask db upgrade
 ```
+========================================================
```
Comment (zeelrupapara) [inline README.md]: remove this line

Comment (ankitjilka) [reply README.md]: I remove

**Review by zeelrupapara — APPROVED** [review]

**Pull request merged**